evrApp/Db/evrscale.db

```diff
@@ -43,3 +43,68 @@ record(waveform, "$(SN)Label-I") {
   field(NELM, "128")
   info(autosaveFields_pass1, "VAL")
 }
+
+record(ai, "$(SN)PhasOffs-SP")
+{
+  field( DESC, "Prescaler $(IDX) Phase Offset")
+  field( EGU , "Deg")
+  field( FLNK, "$(SN)PhasOffs-CO_")
+}
+
+record(calcout, "$(SN)PhasOffs-CO_") {
+  field( DESC, "Degrees to Event Clock Ticks")
+  field( OUT , "$(SN)PhasOffs:Raw-SP PP")
+  field( CALC, "FLOOR(B%360/360*A)")
```

Comment:
The CALC `%` operator maps directly to C `%` operator, which is always
an integer operation.  (as opposed to fmod() from math.h)

https://github.com/epics-base/epics-base/blob/f6d8a1ab6cb685c5ba4e68bf4a5f2e2261e08eed/modules/libcom/src/calc/calcPerform.c#L146

So this will calculation lose precision with larger divisors.

Maybe better to restrict the range of input phases to `[-180, 180)` or `[0,360)`, with appropriate DRVH,DRVL,HOPR,LOPR to communicate/enforce this.

Reply:
Thank you, I pushed a new PR with this taken into account.